formats/placeholder/frontend/schema.json

```diff
@@ -0,0 +1,147 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "base_path",
+    "format",
+    "locale"
+  ],
+  "properties": {
+    "base_path": {
+      "type": "string"
+    },
+    "title": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "format": {
+      "type": "string",
+      "enum": [
+        "placeholder"
```

Comment:
This isn't a new precedence, there is already be a bunch of content items with the "placeholder_FORMAT"-style of format. For example, see https://github.com/alphagov/whitehall/blob/master/app/presenters/publishing_api_presenters/placeholder.rb